modules/apps/portal/portal-upgrade-test/src/testIntegration/resources/META-INF/test-portal-hbm.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://hibernate.sourceforge.net/hibernate-mapping-3.0.dtd">
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
```

Comment:
@yuhai Does Hibernate 5 still use `hibernate-mapping-3.0.dtd`? They only changed the link?

Reply:
@yuhai OK I can confirm that 3.0 is the version for current Hibernate.